src/ui/assets/AssetsUpload.js

```diff
@@ -2,66 +2,72 @@ import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
 import { useDropzone } from 'react-dropzone';
 import { FormattedMessage } from 'react-intl';
-import { Button } from 'patternfly-react';
 
 import UploadAssetModalContainer from 'ui/assets/modals/upload-assets/UploadAssetModalContainer';
 
-const AssetsUpload = ({ onDrop, buttonVersion }) => {
+const AssetsUpload = ({
+  onUpload, onAssetSelected, name, customClassName,
+  customTriggerComponent, customDropzoneProps,
```

Comment:
why wee need those `customDropzoneProps`?

Reply:
Because when a button is used, we need to give dropzone props like this to function: `{ noClick: true, noKeyboard: true }`

Reply:
Is keyboard really a problem so we set `noKeyboard` to true? It's a potential accessibility issue.

Reply:
I followed official docs. Will test without that now and report how it goes here

Reply:
noKeyboard removed. noClick is needed, otherwise button upload clicking has weird behaviour (file browser popping up multiple times)

Reply:
If you think makes sense keeping `noClick` configurable (and so worth prop passing), then I'm good :-)

Reply:
default dropzone needs noClick: false, and button version needs noClick: true, so I guess its needed 👍

Reply:
correct, thanks!